pyproject.toml

```diff
@@ -1,41 +1,38 @@
 [tool.poetry]
 name = "cohere"
-version = "4.34"
```

Comment:
should we major version if we're breaking ?

Reply:
It seems it's pretty standard to consider dropping support for EOL python versions to not be a breaking change (cf https://github.com/PyCQA/isort/releases -- I noticed it when upgrading black/isort/autoflake/... which have all done this) I agree it's not super clear though